### PR TITLE
CCCT-398 || Acknowledge Payment Received

### PR DESCRIPTION
**Pull request by j13panchal**
## Summary
https://dimagi.atlassian.net/browse/CCCT-398

## Safety Assurance

- [ ] If the PR is high risk, "High Risk" label is set
- [ ] I have confidence that this PR will not introduce a regression for the reasons below
- [ ] Do we need to enhance manual QA test coverage ? If yes, "QA Note" label is set correctly

### CODE REVIEW

File: app/res/values-fr/strings.xml [hunk 1]
```diff
@@ -268,4 +268,6 @@ License.
     <string name="connect_button_logged_in">Allez dans le menu Connecter</string>
     <string name="connect_db_corrupt">Un problème est survenu avec la base de données, veuillez récupérer votre compte.</string>
     <string name="connect_job_list_api_failure">Une erreur s\'est produite lors de la connexion au serveur.</string>
+    <string name="connect_payment_acknowledge_notification_yes">Oui</string>
```
Comment (OrangeAndGreen) [inline app/res/values-fr/strings.xml:271]: Good job being so diligent with languages. Can you also add these new keys to the Portuguese translations? Those are actively being used now as well.

File: app/src/org/commcare/google/services/analytics/FirebaseAnalyticsUtil.java [hunk 2]
```diff
@@ -358,6 +358,7 @@ public static void reportCccSignIn(String method) {
                 new String[]{method});
     }
 
+
```
Comment (OrangeAndGreen) [inline app/src/org/commcare/google/services/analytics/FirebaseAnalyticsUtil.java:361]: Extra newline

File: app/src/org/commcare/services/FCMMessageData.java [hunk 3]
```diff
@@ -29,6 +29,7 @@ public FCMMessageData(Map<String, String> payloadData){
         this.creationTime = convertISO8601ToDateTime(payloadData.get("created_at"));
     }
 
+
```
Comment (OrangeAndGreen) [inline app/src/org/commcare/services/FCMMessageData.java:32]: Extra newline

**Review by OrangeAndGreen — APPROVED** [review]
Looks great! I left a few minor edits, a couple nits and a couple translations to add. After that we're ready to merge it in

File: app/res/values-fr/strings.xml (hunk 1 above)
Comment (j13panchal) [reply app/res/values-fr/strings.xml:271]: Done

File: app/src/org/commcare/google/services/analytics/FirebaseAnalyticsUtil.java (hunk 2 above)
Comment (j13panchal) [reply app/src/org/commcare/google/services/analytics/FirebaseAnalyticsUtil.java:361]: Done

File: app/src/org/commcare/services/FCMMessageData.java (hunk 3 above)
Comment (j13panchal) [reply app/src/org/commcare/services/FCMMessageData.java:32]: Done

**Pull request merged**